src/main/java/de/maxhenkel/audioplayer/FilenameMappings.java

```diff
@@ -0,0 +1,87 @@
+package de.maxhenkel.audioplayer;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.util.UUID;
+
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.nbt.ListTag;
+import net.minecraft.nbt.NbtAccounter;
+import net.minecraft.nbt.NbtIo;
+import net.minecraft.server.MinecraftServer;
+
+public class FilenameMappings {
```

Comment:
It is VERY inefficient reading the whole file every time you want to read or save a mapping. Either use caching or keep it in memory and make `FilenameMappings` non-static.